movement/sample_data.py

```diff
@@ -0,0 +1,116 @@
+"""Module for fetching and loading sample datasets.
+
+This module provides functions for fetching and loading sample data used in
+tests, examples, and tutorials. The data are stored in a remote repository
+on GIN and are downloaded to the user's local machine the first time they
+are used.
+"""
+
+from pathlib import Path
+
+import pooch
+import xarray
+import yaml
+
+from movement.io import load_poses
+
+# URL to the remote data repository on GIN
+# noinspection PyInterpreter
+DATA_URL = (
+    "https://gin.g-node.org/neuroinformatics/movement-test-data/raw/master"
+)
+
+# Save data in ¬/.movement/data
+DATA_DIR = Path("~", ".movement", "data").expanduser()
+# Create the folder if it doesn't exist
+DATA_DIR.mkdir(parents=True, exist_ok=True)
+
+# Fetch newest sample metadata
+Path.unlink(DATA_DIR / "poses_files_metadata.yaml", missing_ok=True)
+METADATA_POOCH = pooch.create(
+    path=DATA_DIR,
+    base_url=f"{DATA_URL}",
+    registry={"poses_files_metadata.yaml": None},
+)
+METADATA_PATH = Path(
+    METADATA_POOCH.fetch("poses_files_metadata.yaml", progressbar=True)
+)
```

Comment:
Hi Niko!

1. > Since it's only one file, I think we don't need need to create a `pooch` registry. `pooch` also provides funcitonality for retrieving a single file from a URL, sth like:
    > 
    > ```python
    > metadata_path = pooch.retrieve(
    >     url=f"{DATA_URL}/poses_files_metadata.yaml",
    >     known_hash=None
    > )
    > ```
    Good point. For some reason I was under the impression that it wasn't possible to change the download path from pooch's own (OS-specific) `pooch.os_cache`, so I initially opted for `pooch.create()` + `pooch.fetch()` approach, thinking that manually moving the file from the cache to `.movement/data` after `pooch.fetch()`'ing it would be equally long, and probably harder to follow. However, after having another look at the docs for `pooch.fetch()` I see that I was mistaken! I'll fix this right away.
    
2. > Note that I'm using lowercase here. You picked up my pattern of over-using capitalisation in this file.
    > 
    > How about we unly use capitalisation for `DATA_URL`, `DATA_DIR` (as these are constants that are unlikely to change) and for `SAMPLE_DATA` (to distinguish the download manager from the module itself). I think everything else can be lowercase, what do you think?
    All good! I was indeed just trying to keep a consistent style, but wasn't completely sure what needed to be capitalized and what didn't! I'll change this.